modules/table/src/main/java/org/apache/ignite/internal/table/distributed/BitSetPartitionSet.java

```diff
@@ -19,12 +19,13 @@
 
 import java.util.BitSet;
 import java.util.stream.IntStream;
+import org.apache.ignite.internal.tostring.IgniteToStringExclude;
+import org.apache.ignite.internal.tostring.S;
 
-/**
- * {@link BitSet} implementation of the {@link PartitionSet}.
- */
+/** {@link BitSet} implementation of the {@link PartitionSet}. */
```

Comment:
Please revert it back. I don't get all these useless re-formattings that you do.

Reply:
For me it’s more beautiful, and there are less than 1k lines of changes.